src/lib/DatePicker.svelte

```diff
@@ -29,29 +39,33 @@
     }
     return years
   }
-  $: if (value > max) {
+
+  $: if (value && value > max) {
     setValue(max)
-  } else if (value < min) {
+  } else if (value && value < min) {
```

Comment:
There's a bug here that makes it so users can go to pages beyond the min/max values. Should be fixed if it works like this:
```
  $: if (shownDate > max) {
    updateShownDate(() => max)
  } else if (shownDate < min) {
    updateShownDate(() => min)
  }
```

Reply:
https://github.com/probablykasper/date-picker-svelte/pull/7/commits/941d04e00c5754aaae21df453a3801d18cc9fa73